### PR TITLE
Upgrade to GitLab 8.15.4

### PR DESCRIPTION
**Pull request by phenomax**
Because the other PR (#1046) is still pending, I based the upgrade to v8.15.4 on my uncommited files.
As last reviews show, everything should be fine with v8.15.3 so we can go on and check this PR.

### CODE REVIEW

**Comment by solidnerd** [comment]
https://github.com/sameersbn/docker-gitlab/pull/1051/files#diff-a5e54d1fd7bb69a228ef0dcd2431367e ?

**Comment by phenomax** [comment]
Will remove it tomorrow. Used this file in order to resolve the illegal character in the other commit.

**Comment by phenomax** [comment]
ping @solidnerd please check this PR. I deleted the file :)

**Comment by frol** [comment]
It seems that this PR should be rebased.

**Comment by phenomax** [comment]
@frol Why? A wrongly added file should be no reason for this

File: Changelog.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 This file only reflects the changes that are made in this image. Please refer to the upstream GitLab [CHANGELOG](https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CHANGELOG.md) for the list of changes in GitLab.
 
+**8.15.4
```
Comment (frol) [inline Changelog.md]: It should be wrapped with `**`, so it is parsed as bold, see how it looks now: https://github.com/sameersbn/docker-gitlab/blob/master/Changelog.md

**Comment by frol** [comment]
*Disclaimer*: I am not a maintainer of this project, I just use this image and subscribed to receive the updates.

Rebase is not **required**, but it would be certainly nicer if PR includes only new commits (currently it shows the already merged ones), and no garbage is added/removed for no reason (I refer to `a.txt` file, which will land into the git repo if you leave it this way).

**Comment by phenomax** [comment]
@frol Your right. Because of the wrongly added a.txt, i decided re-creating my pr and rebased the project. Please see #1052

**Pull request closed without merging**